File: app/src/main/java/ua/com/amicablesoft/android/wr/models/Specification.java
package ua.com.amicablesoft.android.wr.models;

/**
 * Created by olha on 2/27/17.
 */

public class Specification {

    private boolean squats;
    private boolean benchPress;
    private boolean deadLift;
    private String powerlifterName;
    private String competition;

    public Specification() {
        squats = true;
        benchPress = true;
        deadLift = true;
        competition = "- None -";
    }

    public boolean isSquats() {
        return squats;
    }

    public void setSquats(boolean squats) {
        this.squats = squats;
    }

    public boolean isBenchPress() {
        return benchPress;
    }

    public void setBenchPress(boolean benchPress) {
        this.benchPress = benchPress;
    }

    public boolean isDeadLift() {
        return deadLift;
    }

    public void setDeadLift(boolean deadLift) {
        this.deadLift = deadLift;
    }

    public String getPowerlifterName() {
        return powerlifterName;
    }

    public void setPowerlifterName(String powerlifterName) {
        this.powerlifterName = powerlifterName;
    }

    public String getCompetition() {
        return competition;
    }

    public void setCompetition(String competition) {
        this.competition = competition;
    }
}

File: app/src/main/java/ua/com/amicablesoft/android/wr/dal/Specification.kt
/*
 * Created by Olha Tymoshenko on 5/24/18 8:31 PM.
 * Copyright (c) 2018. All rights reserved.
 */

package ua.com.amicablesoft.android.wr.dal

class Specification {

    var isSquats: Boolean = false
    var isBenchPress: Boolean = false
    var isDeadLift: Boolean = false
    var powerlifterName: String? = null
    var competition: String? = null

    init {
        isSquats = true
        isBenchPress = true
        isDeadLift = true
        competition = "- None -"
    }
}